# Homework #4
|Author| Stanley A. Baronett|
|--|-------------------------------|
|Created | 10/4/2020|

All references to chapter problems, numbered equations, and page numbers are from _Astrophysics in a Nutshell: Second Edition_ by Dan Maoz.

## Chapter 4, Problem 1

From Eq. 4.19, 

\begin{align}
n_e = \frac{8\pi}{3h^3}p_\textrm{F}^3,
\tag{A}
\end{align}

where $n_e$ is the density and $p_F$ the Fermi momentum of a fully degenerate fermion gas.

### a.

For a nonrelativistic electron gas,

\begin{align}
p_\textrm{F} = \sqrt{2m_eE_\textrm{F}},
\tag{B}
\end{align}

where $m_e$ is the electron mass and $E_\textrm{F}$ the Fermi energy.  Plugging (B) into (A) and solving for $E_\textrm{F}$ yields

\begin{align}
n_e &= \frac{8\pi}{3h^3}(2m_eE_\textrm{F})^{3/2} \\
n_e^{2/3} &= \left( \frac{8\pi}{3h^3} \right)^{2/3} 2m_eE_\textrm{F} \\
E_\textrm{F} &= \boxed{ \left( \frac{3n_e}{\pi} \right)^{2/3} \frac{h^2}{8m_e} }
\tag{C}
\end{align}

### b.

From Eq. 4.26,the electron density of a fully ionized gas composed of a single element is

\begin{align}
n_e = Z\frac{\rho}{Am_p},
\tag{D}
\end{align}

where $Z$ and $A$ are the atomic number and mass number, and $m_p$ is the proton mass.  In a typical white dwarf, $\rho \sim 10^6 \textrm{ g}/\textrm{cm}^3$ and $Z/A \approx 0.5$.  We can find a characteristic $n_e$ as follows:

In [1]:
ZA  = 0.5
rho = 1e6 # g/cm^3
m_p = 1.7e-24 # g
n_e = ZA*rho/m_p

print('𝑛ₑ = {:.2e}'.format(n_e))

𝑛ₑ = 2.94e+29


\begin{equation}
\boxed{n_e \approx 2.94 \times 10^{29}}.
\end{equation}

From Eq. 4.53, the characteristic thermal energy of a single electron in a gas of temperature T is

\begin{align}
E_\textrm{th} = \frac{3}{2}kT.
\tag{E}
\end{align}

Assuming a temperature $T = 10^7 \textrm{ K}$, we can evaluate (E) as follows:

In [2]:
T = 1e7 # K
k = 1.4e-16 # erg/K
Eth = 3*k*T/2

print('𝐸ₜₕ = {:.2e} erg'.format(Eth))

𝐸ₜₕ = 2.10e-09 erg


Plugging our result for $n_e$ into (C), we find:

In [3]:
import numpy as np

h = 6.6e-27 # erg*s
m_e = 9.1e-28 # g
E_F = (3*n_e/np.pi)**(2/3)*h**2/8/m_e

print('𝐸_F    = {:.2e} erg'.format(E_F))
print('𝐸ₜₕ/𝐸_F = {:.2e}'.format(Eth/E_F))

𝐸_F    = 2.57e-07 erg
𝐸ₜₕ/𝐸_F = 8.18e-03


\begin{equation}
\boxed{\frac{E_\textrm{th}}{E_\textrm{F}} \approx 8.18 \times 10^{-3}},
\end{equation}

thus we see electrons inside a white dwarf are indeed degenerate.

## Chapter 4, Problem 2

### a.

The white-dwarf luminosity depends primarily on temperature as 

\begin{align}
L \propto T^4.
\tag{F}
\end{align}

Differentiating $L$ with respect to the total population $N$, using the chain rule, yields

\begin{align}
\frac{dL}{dN} &\sim \frac{dL}{dT}\frac{dT}{dt}\frac{dt}{dN} \\
&\sim 4T^3\frac{dT}{dt}\frac{dt}{dN}.
\end{align}

From Eq. 4.57, the white-dwarf cooling rate can be roughly approximated as $dT/dt \propto T^4$. Thus, ignoring constants,

\begin{align}
\frac{dL}{dN} &\sim T^3T^4\frac{dt}{dN} \\
&\sim T^7\frac{dt}{dN}.
\end{align}

From (F), $T \sim L^{1/4}$. Plugging this in and then reciprocating both sides yields

\begin{align}
\frac{dL}{dN} &\sim L^{7/4}\frac{dt}{dN} \\
\frac{dN}{dL} &\sim L^{-7/4}\frac{dN}{dt}.
\end{align}

If we assume that a constant rate of star formation leads to a constant rate of white-dwarf formation, from stars that complete their evolution, then $dN/dt$ would be constant. Thus,

\begin{equation}
\boxed{ \frac{dN}{dL} \sim L^{-7/4} }.
\tag{G}
\end{equation}

### b.

From Eq. 4.56,

\begin{equation}
L = 4\pi r_\textrm{wd}^2\sigma T_\textrm{E}^4,
\tag{H}
\end{equation}

where $T_\textrm{E}$ is the effective temperature of the white-dwarf photosphere.  Since the Universe has a finite age, the oldest white dwarfs are no cooler than $T_\textrm{E} \sim 4000 \textrm{ K}$.  If the most common white dwarfs have a radius $r_\textrm{wd} \sim 8500 \textrm{ km}$, then the low-luminosity cutoff in the white-dwarf luminosity function can be evaluated as follows:

In [4]:
km    = 1e5     # cm
r_wd  = 8500*km # cm
sigma = 5.7e-5  # erg/cm^2/s/K^4
Teff  = 4000    # K
Lsun  = 3.8e33  # erg/s
L     = 4*np.pi*r_wd**2*sigma*Teff**4 # erg/s

print('𝐿 ~ {:.0e} 𝐿sun'.format(L/Lsun))

𝐿 ~ 3e-05 𝐿sun


\begin{equation}
\boxed{L_\textrm{cutoff} \sim 3 \times 10^{-5} L_\odot}.
\end{equation}